athena-cloudwatch-metrics/src/test/java/com/amazonaws/athena/connectors/cloudwatch/metrics/MetricDataQuerySerDeTest.java

```diff
@@ -24,47 +24,48 @@
 import org.slf4j.LoggerFactory;
 import software.amazon.awssdk.services.cloudwatch.model.Dimension;
 import software.amazon.awssdk.services.cloudwatch.model.Metric;
+import software.amazon.awssdk.services.cloudwatch.model.MetricDataQuery;
 import software.amazon.awssdk.services.cloudwatch.model.MetricStat;
 
 import java.util.ArrayList;
 import java.util.List;
 
 import static org.junit.Assert.*;
 
-public class MetricStatSerDeTest
+public class MetricDataQuerySerDeTest
 {
-    private static final Logger logger = LoggerFactory.getLogger(MetricStatSerDeTest.class);
```

Comment:
any way we can keep the expected serialization static? The reasoning why is that the serializer/deserializer has some additional custom logic and only comparing the SerDe roundtrip could hide some issues if there ever was a bad commit. I do think the serDe roundtrip should still exist though as a separate test